src/js/modules/infragistics.util.js

```diff
@@ -55,7 +56,7 @@
 
 	// The base Class implementation (does nothing) or expects Class to already be defined as a function
 	// K.D. August 18, 2016 Bug #242 global scope Class object is overridden by Ignite UI for jQuery Class object
-	this.Class = this.Class || function () { };
+	window.Class = window.Class || function () { };
```

Comment:
I am pretty sure these two lines are not identical and using window here would cause some issue. What I am not sure about is if it weren't something related to Webpack and/or SystemJS and how we should test it (if we even care) ... I'll need some time to get this in the closed source and produce some test scenarios.

Reply:
Without strict mode `this` in anonymous functions is the global object - in this case `window`. With strict mode, this is `undefined` in an anonymous function.